src/Microsoft.DotNet.Interactive.FSharp/FSharpHtml.fs

```diff
@@ -0,0 +1,511 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Interactive.FSharp.FSharpKernelHelpers
+
+open Microsoft.AspNetCore.Html
+open Microsoft.DotNet.Interactive.Formatting
+
+module Html = 
+
+    type internal HtmlElement =
+        | Tagged of PocketView
+        | Text of string
+        | Obj of obj
+
+        override x.ToString() = 
+            match x with 
+            | Tagged p -> p.ToString()
+            | Text s -> s
+            | Obj o -> o.ToString()
+
+        interface IHtmlContent with
+            member p.WriteTo(writer, encoder) = 
+                match p with 
+                | Text s -> encoder.Encode(writer, s)
+                | Tagged  p -> p.WriteTo(writer, encoder)
+                | Obj o -> encoder.Encode(writer, string o)
+
+    type HtmlAttribute = string * obj
+
+    [<AutoOpen>]
+    module HtmlElements =
+        type internal IMarker = interface end
+        let private f name (attrs: HtmlAttribute list) (content: IHtmlContent list) = 
```

Comment:
could we call ```f``` something like ```htmlElement``` or anything that could make it easier for contributors to identify?

Reply:
Will do.  I will also remove the tags "html" and "body" which we never use in this context to reduce namespace pollution

Reply:
Thanks, done